Debug App/Sources/View Controllers/MerchantDropInUIViewController.swift

```diff
@@ -81,72 +88,52 @@ class MerchantDropInUIViewController: UIViewController, PrimerDelegate {
         }
     }
     
-    @IBAction func checkoutWithApplePayButtonTapped(_ sender: Any) {
+    @IBAction func showPaymentMethodButtonTapped(_ sender: Any) {
+        guard let paymentMethod = paymentMethodTypeTextfield.text else {
+            return
+        }
+        
         print("\n\nMERCHANT APP\n\(#function)\n")
         self.logs.append(#function)
         
         if let clientToken = clientToken {
-            Primer.shared.showPaymentMethod("APPLE_PAY", intent: .checkout, clientToken: clientToken)
-            
+            Primer.shared.showPaymentMethod(paymentMethod,
+                                            intent: paymentMethodTypeSessionIntent,
+                                            clientToken: clientToken)
         } else if let clientSession = clientSession {
             Networking.requestClientSession(requestBody: clientSession) { (clientToken, err) in
                 if let err = err {
                     print(err)
                     let merchantErr = NSError(domain: "merchant-domain", code: 1, userInfo: [NSLocalizedDescriptionKey: "Failed to fetch client token"])
                     print(merchantErr)
                 } else if let clientToken = clientToken {
-                    Primer.shared.showPaymentMethod("APPLE_PAY", intent: .checkout, clientToken: clientToken)
+                    Primer.shared.showPaymentMethod(paymentMethod,
+                                                    intent: self.paymentMethodTypeSessionIntent,
+                                                    clientToken: clientToken)
                 }
             }
-        } else {
-            fatalError()
         }
     }
     
-    @IBAction func vaultKlarnaButtonTapped(_ sender: Any) {
-        print("\n\nMERCHANT APP\n\(#function)\n")
-        self.logs.append(#function)
-        
-        if let clientToken = clientToken {
-            Primer.shared.showPaymentMethod("KLARNA", intent: .vault, clientToken: clientToken)
-            
-        } else if let clientSession = clientSession {
-            Networking.requestClientSession(requestBody: clientSession) { (clientToken, err) in
-                if let err = err {
-                    print(err)
-                    let merchantErr = NSError(domain: "merchant-domain", code: 1, userInfo: [NSLocalizedDescriptionKey: "Failed to fetch client token"])
-                    print(merchantErr)
-                } else if let clientToken = clientToken {
-                    Primer.shared.showPaymentMethod("KLARNA", intent: .vault, clientToken: clientToken)
-                }
-            }
-        } else {
-            fatalError()
+    
+    @IBAction func paymentMethodSessionIntentChanged(_ sender: UISegmentedControl) {
```

Comment:
We haven't included the `KLARNA` method from above in this - was it just not needed or is to be reintroduced?